BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.h

```diff
@@ -86,6 +86,8 @@ typedef NS_OPTIONS(NSUInteger, BOXRepresentationRequestOptions) {
 
 - (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)representationOptions args:(va_list)args;
```

Comment:
Can we remove the header of the replaced method?

Reply:
yes, it was a miss. good catch